src/libraries/System.Runtime/tests/System.Runtime.Tests/ILLink.Descriptors.iOS.xml

```diff
@@ -0,0 +1,40 @@
+<linker>
+  <assembly fullname="TestLoadAssembly" />
+  <assembly fullname="TestILAssembly" />
+  <assembly fullname="mscorlib" />
+  <assembly fullname="System.Threading.Overlapped" />
+  <assembly fullname="System.Reflection.TestModule" />
+  <assembly fullname="System.TestStructs" />
+  <assembly fullname="System.Runtime.Tests" >
+    <type fullname="System.Tests.EnumTests" />
+    <type fullname="System.Tests.ArrayTests" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.Nullable`1[[System.Int32,System.Private.CoreLib]],System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.Byte, System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.SByte, System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.UInt16, System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.Int16, System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.UInt32, System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.Int32, System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.UInt64, System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.Int64, System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.IntPtr, System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.UIntPtr, System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.Char, System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.Boolean, System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.Single, System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.Double, System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.Decimal, System.Private.CoreLib]]" />
+    <type fullname="System.Reflection.Tests.InvokeRefReturnNetcoreTests+TestClass`1[[System.Reflection.BindingFlags, System.Private.CoreLib]]" />
```

Comment:
I don't think we need this and we should just root the generic type (not the instantiations).